atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/service/TransactionService.java

```diff
@@ -52,19 +52,19 @@ public interface TransactionService extends AutoCloseable, AsyncTransactionServi
     Map<Long, Long> get(Iterable<Long> startTimestamps);
 
     /**
-     * This operation is guaranteed to be atomic and only set the value if it hasn't already been
-     * set.
-     * @throws KeyAlreadyExistsException If this value was already set, but {@link #get(long)} should
-     * be called to check what the value was set to.  This may throw spuriously due to retry.
+     * This operation is guaranteed to be atomic and only commit the value if it hasn't already been
```

Comment:
Note that this will break large internal product, and also atlasdb-proxy (I didn't check for other users, but `txMgr.getTransactionService()` is definitely a thing we expose and a thing people can use).

Reply:
ok will be reverting this change and modifying the docs